### DISCUSSION TITLE
번역 안된 피규어 인형 3개

### DISCUSSION THREAD

**Discussion by Rirbe** [wb1016/sb_korpatch#47] [unfinished]
Cosmic Intruder Figurine
우주 침입자 피규어
The label reads, "An entity born outside of physical space and time."
라벨에는 "물리적 공간과 시간 밖에서 태어난 실체." 라고 적혀있다.

Parasprite Figurine
para+sprite 인거 같은데 뇌사도깨비 라고 해야할지 고민입니다.

The label reads, "A braindead bioweapon created by irresponsible human scientist."
라벨에는 "무책임한 인간 과학자가 만든 뇌사 생물무기."라고 적혀있다.

Swansong Figurine
(스완송)백조의 노래 피규어(메카 이름이라 스완송으로 갈지 백조의 노래가 좋을지..) 
The label reads, "An experimental mech suit, built and piloted by Asra Nox."
라벨에는 "아스라 녹스에 의해 만들어지고 조종된 실험용 메카 슈트"라고 적혀 있다.

**Comment by OABHs** [comment]
주말이라 잠시 짬이 나서 찾아봤습니다


**entity**의 번역의 경우 실체라는 단어를 생물에게 사용할 때에는 그것이 하나의 존재라는 의미보단 본질을 가리킬때 사용합니다.

따라서 실체라고 하기보다는, 인게임 상 사이버스페이스에 존재하는 디지털 공간에서 등장하므로

생물을 지칭할 수도 있고, 프로그래밍과 IT 분야에서도 사용할 수 있는 '**개체**'가 적절할 것 같습니다.



para - 의 의미가 

1. beyond 무언가를 넘어섰거나 지나친 
2. 준 내지는 보조? 자기와 비슷한 전문직, 전문 조직을 돕는? ex) parmedic 
3. 이상, 결점
4. 옆에있거나 평행하는 
5. 함께하는 ex) parasite

등이 있는데
보통 2.은 직업이나 조직의 맥락, 3.은 주로 의료의 맥락에서 기능적으로 문제가 있을 때, 4.의 경우는 꼭 그런건 아니지만 과학, 화학 영역에서 자주 사용됩니다.

sprite는 말씀하신 대로 도깨비도 가능하고 요정, 정령, 유령, 영혼 등 초자연적이거나 영적인 존재를 나타내는 말들로 번역이 가능합니다.

저는 로어상 Miniknog와 USCM의 생물학 실험으로 탄생한 괴물이니 3. 뜻을 의도하고 붙인 이름이라 생각합니다.

그렇다면 '**결함이 있는 초자연적 존재**'의 의미를 가진 번역명이 적절하다는 건데, '이상한 도깨비' 라던가 '이상한 유령'정도를 떠올려 봤는데, '이상한'도 그렇고 '도깨비'도 그렇고 제가 볼때는 영 어색합니다. 

개인적으로는 그냥 음차하고 싶습니다만, 꼭 완역해야한다면 저 의미를 안어색하게 살려서 번역하거나 뇌사도깨비처럼 특징에서 따와서 의역을 아주 잘해야 될 것 같습니다.

그리고 혹시 다른 뜻을 사용하게된다면
1.의 경우 뒤에 올 sprite의 느낌을 더 강하게 해준다고 봐야할것이고(초월)
2. 4. 은 고려 X
5.의 경우 parasite마냥 '기생 유령' 정도의 의미를 가질 것 같습니다.

자세한 로어도 더 공개되어있지 않고, 인게임에서는 몸통박치기 외에는 하지 않으니 막막하네요.

해당 몬스터의 외형은 https://starbounder.org/Parasprite 나오는데 뿔이 3개 달려있으며 눈은 초점없이 계속 움직이고 입도 계속 닫았다가 벌렸다가 합니다. 기본적으로 공중에서 부유하며 벽은 통과하지 못하고 플레이어를 발견하면 계속 몸통박치기를 합니다.



스완송의 경우 **최후의 작품/공연/경기**의 의미를 가집니다(보통 걸작이거나 아름다운 끝맺음 등 긍정적인 이미지)

백조의 노래도 맞는 말이긴 한데, 우리나라에는 '백조의 마지막 노래가 아름답다'라는 관용구가 없으니 가능하면 그 진짜 의미를 살려 번역하는 편이 좋을 것 같습니다.
